Makefile.PL

```diff
@@ -8,6 +9,11 @@ use Crypt::OpenSSL::Guess qw(openssl_inc_paths openssl_lib_paths);
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
 
+my $libs = ' -lssl -lcrypto';
+if ( $Config{osname} eq 'aix' ) {
+  $libs = $libs . ' -lz';
```

Comment:
This kinda looks like it might fit nicely inside [`Crypt::OpenSSL::Guess::openssl_lib_path`](https://metacpan.org/dist/Crypt-OpenSSL-Guess/source/lib/Crypt/OpenSSL/Guess.pm#L43)?

:warning: I'm not the maintainer.